x-pack/plugins/event_log/scripts/mappings.js

```diff
@@ -207,6 +219,7 @@ exports.EcsPropertiesToGenerate = [
   'rule',
   'user.name',
   'kibana',
+  'action',
```

Comment:
once the `action` property is moved to `kibana`, you can delete this line ...

Reply:
Resolved in this commit f933a09f2240f33b542881dcb715e6c7b4a645dc